### ISSUE TITLE
[MAUI][Regression] The recipe details page is not fully displayed

### ISSUE THREAD

**Issue by jaosnz-rep** [dotnet/maui#15872]
### Description

The recipe details page is not fully displayed

### Steps to Reproduce

1.Deploy https://github.com/rachelkang/recipeSearch to Windows
2.Tap or click on the "Breakfast" button.
3.Tap or click on the first recipe in the CollectionView, "Breakfast Sausage Puffs".

Actual results:
Bottom button display is truncated.
![image](https://github.com/dotnet/maui/assets/82628041/16353b02-1b6a-4600-91ad-2bfe10f5fbd7)

### Link to public reproduction project repository

https://github.com/rachelkang/recipeSearch

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

net7.0-windows10.0.19041.0

**Comment by Redth** [comment]
I don't think this is an actual regression.  I tested on both 7.0.86 and 7.0.90 and the results look the same.  If you shrink the window short enough vertically the button starts clipping due to how the layout of the page is written.

# 7.0.86

https://github.com/dotnet/maui/assets/271950/bf25d334-3e90-4235-8378-c23634a83388


------

# 7.0.90

https://github.com/dotnet/maui/assets/271950/b4408d17-29a7-45ba-9870-b348e1eb5fd7

**Comment by jaosnz-rep** [comment]
This issue is not reproduced on 17.9 P4 using 8.0.7-nightly.9926+sha.cb1ac24b9a-azdo.8974658.
